Amplitude.js

```diff
@@ -264,6 +270,10 @@
                     ampSettings.includeReferrer = forwarderSettings.includeReferrer == 'True';
                 }
 
+                if (forwarderSettings.forceHttps) {
+                    ampSettings.forceHttps = forwarderSettings.forceHttps;
```

Comment:
do you need something like the above for includeReferrer, since this will be a string?